src/methods/validators.rs

```diff
@@ -1,3 +1,64 @@
+//! Queries active validators on the network.
+//!
+//! Returns details and the state of validation on the blockchain.
+//!
+//! ## Examples
+//!
+//! - Get the validators for a specified epoch.
+//!
+//!     ```
+//!     use near_jsonrpc_client::{methods, JsonRpcClient};
```

Comment:
what I mean

```suggestion
//! - Get the validators for a specified epoch.
//!
//!   ```
//!   use near_jsonrpc_client::{methods, JsonRpcClient};
```